addons/payment/models/payment_acquirer.py

```diff
@@ -350,7 +350,7 @@ def _should_build_inline_form(self, is_validation=False):
         """
         return True
 
-    def _compute_fees(self, amount, currency, country):
+    def _compute_fees(self, amount, currency, country, operation=None):
```

Comment:
If there is only one place where the operation is given, You should only manage it there (in the tx creation), without modifying this method.

Reply:
fixed it